rules/rules.js

```diff
@@ -22,8 +23,8 @@
  * @property {string} oldStatus only for {@link triggers.ThingStatusChangeTrigger}: Previous state of Thing that triggered event
  * @property {string} newStatus only for {@link triggers.ThingStatusChangeTrigger}: New state of Thing that triggered event
  * @property {string} status only for {@link triggers.ThingStatusUpdateTrigger}: State of Thing that triggered event
- * @property {string} eventType for all triggers except {@link triggers.PWMTrigger}, {@link triggers.PIDTrigger}: Type of event that triggered event (change, command, time, triggered, update)
- * @property {string} triggerType for all triggers except {@link triggers.PWMTrigger}, {@link triggers.PIDTrigger}: Type of trigger that triggered event (for `TimeOfDayTrigger`: `GenericCronTrigger`)
+ * @property {string} eventType for all triggers except {@link triggers.PWMTrigger}, {@link triggers.PIDTrigger}, time triggers: Type of event that triggered event (change, command, time, triggered, update)
```

Comment:
when we say "time triggers", do we mean `TimeOfDayTrigger` and `DateTimeTrigger`?  If so, should we just call those out directly ?

Reply:
We mean all three time triggers that we have:
- `TimeOfDayTrigger`
- `GenericCronTrigger`
- `DateTimeTrigger`